proprietary/design-tokens/build/build-tokens.mjs

```diff
@@ -0,0 +1,242 @@
+import { permutateThemes as permutateTsThemes, registerTransforms } from '@tokens-studio/sd-transforms';
+import { transform } from '@divriots/style-dictionary-to-figma';
+import { readFile } from 'node:fs/promises';
+import StyleDictionary from 'style-dictionary';
+import { IMPORTED_SRC_FOLDER, DIST_FOLDER } from './constants.mjs';
+import jsonListFormat from './json-list-formatter.mjs';
+
+/** 
+ * Transforms spacing tokens with math symbols to css calc()-values
+ * NOTE: because the way references work in StyleDictionary/TokenStudio it will not transform e.g. `5 * {space.cowboy}`
+ */
+StyleDictionary.registerTransform({
+  name: 'spacing/calc',
+  type: 'value',
+  transitive: true,
+  matcher: function(token) {
+    const {type, value, isSource} = token
+    return type === 'spacing' && isSource && value.match(/[\+\-\*\/]/g);
+  },
+  transformer: function(token) {
+    return `calc(${token.original.value})`;
+  }
+})
+
+registerTransforms(StyleDictionary, {});
```

Comment:
Ik bedoelde dus dat dit hele blokje niet nodig is. Als dat om e.o.a. reden wel nodig is moet je daar een comment bij plaatsen waarom dat zo is.